ministryofjustice/laa-review-criminal-legal-aid
# Update error copy, add nav to internal error pages

## Description of change
Updated copy for existing flash messages and error pages.
Added navigation to internal error pages so users can get somewhere useful. 

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-336

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/1057f197-026c-491e-b8e6-482a17e577e0)

![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/4801c6fa-7900-4a76-a2e2-7d1b055c9670)

![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/beea675a-a301-447c-815c-0b0e35605940)

### After changes:
![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/6a52ca4f-9c4c-409b-8753-38a39e803163)

![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/66f75ec5-66ac-4ecd-be97-d20ac118cb18)

![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/a2bb5420-ea64-4cce-a6ea-a10ef3a424c1)

## How to manually test the feature
To see service errors (not get redirected to user admin) set feature flag in config/settings.yml:
```
allow_user_managers_service_access:
    local: true
```
Doing so will make authorisation_spec and header_nav_spec fail
To see all error pages, remove from routes.rb:

```
, constraints:
    lambda { |_request| !Rails.application.config.consider_all_requests_local }
```
